services/webhook/tests/test_rerun.py

```
@@ -1486,3 +1486,52 @@ def test_defuse_md_breaks_mentions():
+
+_ZWSP = chr(0x200B)  # kept as a runtime chr() call, not a literal
+# character, so this source file stays 7-bit ASCII on disk.
+
+
+def test_run_ask_neutralizes_mention_in_posted_answer(monkeypatch):
+    """A prompt-injected diff could get the model to include a live
+    @mention in its answer; posted under Grug's own installation-token
+    authority, that would notify a real GitHub user as if Grug itself
+    pinged them (#561, same class as Teller's #554 round-4 fix)."""
+    posted = []
+    monkeypatch.setattr(rerun, "with_install_token_retry", lambda iid, fn: fn("tok"))
```

Ruff (0.15.21)

[warning] Unused lambda argument: `iid`

(ARG005)